sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.7.0 (Unreleased)
+## 1.7.0 (2022-03-21)
```

Comment:
"Breaking Changes" section below is empty which will fail CI at the very last step.

Reply:
ohhh thank you! I always forget the empty sections